libs/sdk-core/src/lsp.rs

```diff
@@ -51,33 +52,42 @@ impl LspInformation {
             lsp_pubkey: lsp_info.lsp_pubkey,
             max_inactive_duration: lsp_info.max_inactive_duration,
             channel_minimum_fee_msat: lsp_info.channel_minimum_fee_msat,
-            opening_fee_params_menu: lsp_info
-                .opening_fee_params_menu
-                .into_iter()
-                .map(|ofp| ofp.into())
-                .collect::<Vec<OpeningFeeParams>>(),
-        }
+            opening_fee_params_menu: OpeningFeeParamsMenu::try_from(
+                lsp_info
```

Comment:
This looks a lot better!
Perhaps also it is better to change the try_from to get the grpc values and move the mapping into our model inside that function?

Reply:
Done